core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMNode.kt

```diff
@@ -32,18 +38,42 @@ data class STDCMNode(
 ) : Comparable<STDCMNode> {
 
     /**
-     * Defines the estimated better path between 2 nodes, in terms of total run time, then departure
-     * time, then number of reached targets. If the result is negative, the current node has a
-     * better path, and should be explored first. This method allows us to order the nodes in a
-     * priority queue, from the best path to the worst path. We then explore them in that order.
+     * Defines the estimated better path between 2 nodes, in the following priority
+     * - lowest total run time
+     * - earliest departure time
+     * - closest planned arrival time, using the current then the last planned timing data
```

Comment:
This should actually be relative to the tolerance on either side. If there a 10 minutes tolerance before and 30 minutes tolerance after, then being late by 2 minutes is better than being early by 1 minute.

Reply:
Good catch.